x-pack/plugins/reporting/server/export_types/common/validate_urls.ts

```diff
@@ -22,6 +22,15 @@ const isBogusUrl = (url: string) => {
 };
 
 export const validateUrls = (urls: string[]): void => {
+  if (!Array.isArray(urls)) {
+    throw new Error('Invalid relativeUrls. String[] is expected.');
+  }
+  urls.forEach((url) => {
+    if (typeof url !== 'string') {
+      throw new Error('Invalid Relative URL in relativeUrls. String is expected.');
+    }
+  });
+
```

Comment:
Factored this out of the compatibility shim: https://github.com/elastic/kibana/blob/57af846/x-pack/plugins/reporting/server/export_types/printable_pdf/create_job/compatibility_shim.js#L67